src/VisualStudio/Core/Def/Implementation/Interactive/CSharpResetInteractiveMenuCommand.cs

```diff
@@ -23,7 +23,9 @@ internal sealed class CSharpResetInteractiveMenuCommand
 
         protected override CommandID GetResetInteractiveFromProjectCommandID()
         {
-            return new CommandID(ID.InteractiveCommands.CSharpInteractiveCommandSetId, ID.InteractiveCommands.ResetInteractiveFromProject);
+            // TODO: Revert to the code below pending https://github.com/dotnet/roslyn/issues/8927 .
+            // return new CommandID(ID.InteractiveCommands.CSharpInteractiveCommandSetId, ID.InteractiveCommands.ResetInteractiveFromProject);
```

Comment:
Not a fan of commenting out code, that's what history is for. Priorities and things changes - if we're shipping with this, we should make the change, and put this commit in the bug.